zowe_conformance/terms_and_conditions.md

```diff
@@ -67,16 +85,18 @@ If The Linux Foundation confirms to the Participant that the tests have been su
 
 The participant must continue to abide by the terms and conditions of the Conformance Program throughout the period of the Participant's use of the Program Marks. This includes all of the terms set forth in this agreement, including without limitation the following specific requirements:
 
-> *Conformance Test Versions*: Conformance for a Qualifying Offering is tied to a specific version of the corresponding conformance tests and evaluation criteria for one or more particular Components. The tests' and evaluation criteria version number and the corresponding Component must be specified within the Conformance Mark logos, as described in the Visual Branding Guide. When a new version of the Test Evaluation Guide is released, conformant applications and services must demonstrate that they pass the updated tests and meet the updated evaluation criteria within 60 days. Applications that do not do so within the allotted time must not be marked as conformant thereafter. A previously-Qualifying Offering can re-conform to the new Test Evaluation Guide to regain conformance at any time by following the steps described above.
+>
+> *Conformance Test Versions*: Conformance for a Qualifying Offering is tied to a specific version of the corresponding conformance tests and evaluation criteria for one or more particular Components. The tests' and evaluation criteria version number and the corresponding Component must be specified within the Conformance Mark logos, as described in the Visual Branding Guide. A previously-Qualifying Offering for a previous version can re-conform to the new Test Evaluation Guide to gain conformance to a newer version at any time by following the steps described above.  Being conformant to a version of Zowe does not roll over to conformance to later versions of Zowe, nor is conformance removed from a version of Zowe when a newer version is released, notwithstanding scenarios described in [Reported Non-conformance](#reported-non-conformance) and [Termination](#termination).
```

Comment:
For this section, let's go with the following text. I think this does what you're looking to accomplish here, by just saying that the version number for conformance needs to be referenced and that they can only use versions for which they've actually confirmed as conformant:

> *Conformance Test Versions*: Conformance for a Qualifying Offering is tied to a specific version of the corresponding conformance tests and evaluation criteria for one or more particular Components. The tests' and evaluation criteria version number and the corresponding Component must be specified within the Conformance Mark logos, as described in the Visual Branding Guide. A Qualifying Offering may only use the Conformance Mark logos and describe itself as conformant with regards to the specific version(s) of conformance for which it qualified.

Reply:
Reads much better - thanks you.  Updated the Conformance Test Versions para